docs/src/main/asciidoc/mp/testing/testing-ng.adoc

```diff
@@ -69,6 +69,14 @@ I.e. By default, the test instance is re-used between test methods.
 
 NOTE: The test instance is not re-used between CDI container, using a dedicated CDI container implies a new test instance
 
+=== Using meta-annotations
+`@HelidonTest` can be used in a meta-annotation:
```

Comment:
Update the text to something like this:

Meta-annotations are supported on both test classes and test methods  and can be used as a composition mechanism.